database/migrations/default/1744126149551_noncr3_drop_date_hour_cols/up.sql

```diff
@@ -0,0 +1,346 @@
+--
+-- add a timestamptz column
+--
+alter table atd_apd_blueform add column case_timestamp timestamptz;
+comment on column atd_apd_blueform.case_timestamp is 'The timestamp at which the APD case occurred';
+
+--
+-- backfill the timestamptz based on date and hour columns
+-- this is a one-time backfill—we will also import a backfill with
+-- full-resolution timestamps through the VZE UI
+--
+update atd_apd_blueform set
+    case_timestamp = (date::text || ' ' || lpad(hour::text, 2, '0') || ':00:00')::timestamp at time zone 'America/Chicago';
+
+--
+-- add fk column to atd_apd_blueform.case_id
+--
+alter table ems__incidents add column atd_apd_blueform_case_id integer;
+alter table ems__incidents
+add constraint ems__incidents_atd_apd_blueform_case_id_fkey foreign key (
+    atd_apd_blueform_case_id
+)
+references public.atd_apd_blueform (
+    case_id
+) on update cascade on delete set null;
+
+comment on column ems__incidents.atd_apd_blueform_case_id is 'The non-CR3 case ID matched to this record';
+
+--
+-- add timestamp audit fields 
+--
+alter table atd_apd_blueform add column created_at timestamptz not null default now();
+alter table atd_apd_blueform add column updated_at timestamptz not null default now();
+create trigger set_public_atd_apd_blueform_updated_at before update on public.atd_apd_blueform for each row execute function public.set_current_timestamp_updated_at();
+
+
+--
+-- add soft delete column
+--
+alter table atd_apd_blueform add is_deleted boolean not null default false;
+
+comment on column atd_apd_blueform.is_deleted is 'Indicates soft-deletion';
+
+--
+-- add some indexes
+--
+create index atd_apd_blueform_case_timestamp_index on public.atd_apd_blueform (
+    case_timestamp
+);
+create index ems__incidents_atd_apd_blueform_case_id_index on public.ems__incidents (
+    atd_apd_blueform_case_id
+);
+
+create index atd_apd_blueform_is_deleted_index on public.atd_apd_blueform (
+    is_deleted
+);
+
+create index crashes_case_id_investigat_agency_id_idx on public.crashes (
+    case_id, investigat_agency_id
+);
+
+--
+-- Trigger which soft-deletes atd_apd_blueform records if there is a cr3 crash
+-- with the same case_id and investigat_agency_id is 74 (Austin PD)
```

Comment:
and if they occurred within a 12 hour range of each other? could add that detail to this comment